Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook

```diff
@@ -525,7 +597,7 @@
             "name": "FinalQuery",
             "type": 1,
             "isRequired": false,
-            "query": "print(\r\niff({Hierarchy} == 0, \"{Computer_Process_IP_Port}\", \r\niff({Hierarchy} == 1, \"{Computer_Process_IP}\",\r\niff({Hierarchy} == 2, \"{Computer_Process}\",\r\niff({Hierarchy} == 3, \"{Computer}\", \"{Computer}\")))));",
+            "query": "print(strcat(\r\niff({Hierarchy} == 0, \"{Computer_Process_IP_Port}\", \r\niff({Hierarchy} == 1, \"{Computer_Process_IP}\",\r\niff({Hierarchy} == 2, \"{Computer_Process}\",\r\niff({Hierarchy} == 3, \"{Computer}\", \"{Computer}\")))),' | where MaliciousConnectionsCount >= {OnlyMaliciousConnections}',' | where LinksFailed >= {OnlyLinksFailed}',' | where AverageResponseTime >= {HighResponseTime}'));",
             "crossComponentResources": [
```

Comment:
For HighResponseTime, once we have tool tips add info saying > 1sec.

Reply:
The new dropdown looks like this, so should be self-explanatory for the time being
![image](https://user-images.githubusercontent.com/43890980/52826201-c15dcb80-3074-11e9-9586-3c46272ae1ac.png)